twins/nats/senml.go

```diff
@@ -0,0 +1,22 @@
+package nats
+
+// Record represents one senML record.
+type Record struct {
```

Comment:
You have this in mainflux/senml lib no ? Maybe is better idea to use structure from lib, where you have validation and everything not just copy/paste it.

Reply:
Yes, at the moment of coding, it was not there. Will use from `mainflux/senml`.

Reply:
Fixed.